source/MapPanel.cpp

```diff
@@ -218,13 +218,14 @@ void MapPanel::Draw()
 
 	// Draw the "visible range" circle around your current location.
 	Color dimColor(.1f, 0.f);
```

Comment:
We should move to defining these colors in the interface file while we're here.

Reply:
will get to it

Reply:
done